Source/svg/SVGParser.swift

```diff
@@ -1571,6 +1574,13 @@ private class PathDataReader {
         return current
     }
 
+    private func isSymbolAcceptable(_ ch: UnicodeScalar?) -> Bool {
+        if let ch = ch {
+            return (ch >= "0" && ch <= "9") || " MLCQAzHVSTmlcqahvstEe,".contains(String(ch))
```

Comment:
You need to do these checks carefully. Part of this already done by `isNumStart` and `getPathSegmentType` methods, so you don't need to check it twice.

Reply:
And where did you find this list of acceptable symbols? It looks incomplete for me.